aten/src/ATen/native/cuda/MaxMinElementwiseKernel.cu

```diff
@@ -62,7 +62,33 @@ void minimum_kernel_cuda(TensorIterator& iter) {
   }
 }
 
+void fmax_kernel_cuda(TensorIterator& iter) {
+  if (isFloatingType(iter.common_dtype())) {
+    AT_DISPATCH_FLOATING_TYPES_AND2(at::ScalarType::Half, at::ScalarType::BFloat16, iter.dtype(), "fmax_cuda", [&]() {
+      gpu_kernel_with_scalars(iter, []GPU_LAMBDA(double a, double b) -> scalar_t {
```

Comment:
double -> scalar_t ? or do we need double here?

Reply:
It should be `scalar_t` here, I just made a misunderstand, thanks for correcting!